sdks/java/core/src/main/java/org/apache/beam/sdk/util/ZipFiles.java

```diff
@@ -226,6 +228,8 @@ static void zipDirectory(
    * @throws IOException the zipping failed, e.g. because the input was not
    *     readable.
    */
+  @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE",
```

Comment:
What is the false positive warning? In this case, I feel that findbugs should be warning the caller, as neither of `sourceDirectory` nor `outputStream` are marked `@Nullable`. I'm missing something, I guess?